appveyor.yml

```diff
@@ -0,0 +1,36 @@
+environment:
+
+  global:
+      PYTHON: "C:\\conda"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+      PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
+                        # of 32 bit and 64 bit builds are needed, move this
+                        # to the matrix section.
+
+  matrix:
+    - PYTHON_VERSION: "2.7"
+      CONDA_DEPENDENCIES='pytest qt5 pyqt5' CONDA_CHANNELS='spyder-ide'
+    - PYTHON_VERSION: "2.7"
+      CONDA_DEPENDENCIES='pytest qt pyqt'
+    - PYTHON_VERSION: "2.7"
+      CONDA_DEPENDENCIES='pytest qt pyside'
+    - PYTHON_VERSION: "3.5"
+      CONDA_DEPENDENCIES='pytest qt5 pyqt5' CONDA_CHANNELS='spyder-ide'
+    - PYTHON_VERSION: "3.5"
+      CONDA_DEPENDENCIES='pytest qt pyqt'
+
+platform:
+    -x64
+
+install:
+    - "git clone git://github.com/astropy/ci-helpers.git"
+    - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+    - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+    - "activate test"
+
+# Not a .NET project, we build SunPy in the install step instead
```

Comment:
Ok, I've done this locally - let me know once AppVeyor is enabled and I'll push the changes.